Makefile

```diff
@@ -105,10 +105,10 @@ docker:
 
 .PHONY: run-docker
 run-docker:
-	# To run the Docker container locally first build the Docker image using
-	# `make docker` and then set the environment variables below to appropriate
-	# values. You can find the environment variables needed for development in
-	# .devdata.env (after running `make devdata`) and in tox.ini.
+	# To run the Docker container locally:
+	# 1. make devdata
+	# 2. make docker
+	# 3. make run-docker
```

Comment:
As of this PR it's no longer necessary to edit `Makefile` to get `make run-docker` to work. But we do still have a duplicated list of envvars in `Makefile` that's likely to fall out of date causing `make run-docker` to break